### DISCUSSION TITLE
Preset For Earthly

### DISCUSSION THREAD

**Discussion by idodod** [renovatebot/renovate#26085] [Suggest an Idea]
### Tell us more.

Hi,
I work for [Earthly](https://earthly.dev/) and I'd like to propose adding a new preset to handle Earthly related dependencies.

FWIW Because an Earthfile (An earthly config file) is similar in its structure to a Dockerfile, extending the dockerfile manager's `fileMatch` field to include`Earthfile` is one thing this preset will do.

A couple of more rules will be added for updating the earthly version in its [github action](https://github.com/earthly/actions-setup) and other type of dependencies in an Earthfile that are not docker images.

Side note: independently I hope at some point to offer a new manager separate than dockerfile, which again will be very similar to `dockerfile`.

Thank you for your consideration!

**Comment by secustor** [comment]
You can use also host your presets here on Github and document that for your users.   
Aqua does this for example https://aquaproj.github.io/docs/products/aqua-renovate-config/

Later the `github-actions` manager can be extended and an `earthly` one created.
